### PR TITLE
Add gemini-1.5-pro-002 and gemini-1.5-flash-002

### PR DESCRIPTION
**Pull request by ushuz**
## Title

Add gemini-1.5-pro-002 and gemini-1.5-flash-002

## Type

📖 Documentation
🚄 Infrastructure

## Changes

Add gemini-1.5-pro-002 and gemini-1.5-flash-002

## [REQUIRED] Testing - Attach a screenshot of any new tests passing locall
If UI changes, send a screenshot/GIF of working UI fixes

Supersedes and closes #5874

### CODE REVIEW

**Comment by pedroallenrevez** [comment]
please emrege

File: model_prices_and_context_window.json [hunk 1]
```diff
@@ -1975,6 +1975,32 @@
         "supports_response_schema": true, 
         "source": "https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#foundation_models"
     },
+    "gemini-1.5-pro-002": {
+        "max_tokens": 8192,
+        "max_input_tokens": 1000000,
```
Comment (Manouchehri) [inline model_prices_and_context_window.json:1980]: Shouldn't this be 2097152?

https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#gemini-1.5-pro

File: model_prices_and_context_window.json [hunk 2]
```diff
@@ -2142,6 +2168,38 @@
         "supports_response_schema": true,
         "source": "https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#foundation_models"
     },
+    "gemini-1.5-flash-002": {
+        "max_tokens": 8192,
+        "max_input_tokens": 1000000,
```
Comment (Manouchehri) [inline model_prices_and_context_window.json:2173]: 1048576 I think?

https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#gemini-1.5-flash

**Review by Manouchehri — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Fixed here - https://github.com/BerriAI/litellm/commit/39c9150e975d2ea1218a33349a499d837b288c4f @Manouchehri

**Comment by SmartManoj** [comment]
`gemini/gemini-1.5-flash-8b-exp-0924` is missing

**Comment by ghost** [comment]
@SmartManoj addressed here - https://github.com/BerriAI/litellm/commit/5bc5eaff8a28a1c452bf61e0f3d8f802b7f3e28b

**Pull request merged**